app/controllers/workshop_logs_controller.rb

```diff
@@ -79,7 +70,7 @@ def show
     @answers      = @workshop_log.report_form_field_answers
 
     if @workshop_log
-      if @workshop_log.project && current_user.project_ids.include?(@workshop_log.project.id)
+      if current_user.super_user? || (@workshop_log.project && current_user.project_ids.include?(@workshop_log.project.id))
```

Comment:
this line will need to be updated once we've completed the Organization body of work (projects were previously used for agencies aka organizations)